src/browser/declarativewebutils.cpp

```diff
@@ -92,6 +92,30 @@ void DeclarativeWebUtils::handleDumpMemoryInfoRequest(QString fileName)
     }
 }
 
+void DeclarativeWebUtils::openUrl(QString url)
+{
+
+    QFileInfo fileInfo(url);
+    QUrl targetUrl(url);
+
+    if (targetUrl.scheme().isEmpty()) {
```

Comment:
QUrl::isRelative() does the same thing than .scheme().isEmpty(), right?

Reply:
Nope, doesn't work here as url scheme could be empty. E.g. ../../foobar/mytextfile.txt is relative file path but from url point of view is false.